moveit_core/exceptions/src/exceptions.cpp

```diff
@@ -35,14 +35,18 @@
 /* Author: Acorn Pooley, Ioan Sucan */
 
 #include <moveit/exceptions/exceptions.h>
-#include <ros/console.h>
+#include <rcutils/logging_macros.h>
+#include "rclcpp/rclcpp.hpp"
+
+// Logger
+rclcpp::Logger logger_exceptions = rclcpp::get_logger("exceptions");
 
 moveit::ConstructException::ConstructException(const std::string& what_arg) : std::runtime_error(what_arg)
 {
-  ROS_ERROR_NAMED("exceptions", "Error during construction of object: %s\nException thrown.", what_arg.c_str());
+  RCLCPP_ERROR(logger_exceptions, "Error during construction of object: %s\nException thrown.", what_arg.c_str());
 }
 
 moveit::Exception::Exception(const std::string& what_arg) : std::runtime_error(what_arg)
 {
-  ROS_ERROR_NAMED("exceptions", "%s\nException thrown.", what_arg.c_str());
+  RCLCPP_ERROR(logger_exceptions, "Exception thrown.", what_arg.c_str());
```

Comment:
Why is "%s\n" removed from the error message? This way the exception is not printed to the log

Reply:
That's right, thanks for the catch @henningkayser https://github.com/ros-planning/moveit2/pull/7/commits/b329df8f4a41b061a55a138f03e942bfb02bc1d3